component/src/main/java/org/wso2/carbon/identity/authenticator/linkedIn/LinkedInAuthenticatorConstants.java

```diff
@@ -39,4 +39,6 @@
     public static final String LAST_NAME = "lastName";
     public static final String USER_ID = "id";
     public static final String CLAIM_DIALECT_URI = "http://wso2.org/linkedin/claims";
+    public static final String OAUTH2_PARAM_ERROR = "error";
+    public static final String OAUTH2_PARAM_ERROR_DESCRIPTION = "error_description";
 }
```

Comment:
Shall we add a new line here?

Reply:
The line should be added after L45.

Reply:
Fixed in e8881f23b757064a861cb462417114dde11221e4